src/oc/web/core.cljs

```diff
@@ -150,13 +160,6 @@
     (router/set-route! (vec (remove nil? [route-name org])) {:org org :query-params (:query-params params)}))
   (post-routing)
   (when-not (contains? (:query-params params) :jwt)
-    (when (contains? (:query-params params) :login-redirect)
-      (cook/set-cookie!
-       :login-redirect
-       (:login-redirect (:query-params params))
-       (* 60 60)
-       "/"
-       ls/jwt-cookie-domain ls/jwt-cookie-secure))
```

Comment:
@thepug you removed all this code, did you replace it with something else? We don't use much the login redirect but in some places i think it's still necessary..

Reply:
So I went through mainline and tested all the places I could see login redirect being used and none of it was used. The app just redirects me to the current board after logging in.  I removed all this code because of that.